ansible/playbooks/windows.yml

```diff
@@ -193,24 +206,56 @@
       win_command: C:\Strawberry\perl\bin\perl -MCPAN -e "install Text::CSV_XS"
       tags: text_cvs
 
+    # freemarker v2.3.8
+    - name: Test if freemarker is already installed
+      win_stat:
+        path: 'C:\openjdk\freemarker.jar'
+      register: freemarker_installed
+      tags: freemarker
+
     - name: Download freemarker
       win_get_url: 
         url: http://central.maven.org/maven2/freemarker/freemarker/2.3.8/freemarker-2.3.8.jar
-        dest: 'C:\temp\freemarker.jar'
-        force: no
+        dest: 'C:\openjdk\freemarker.jar'
+      when: (freemarker_installed.stat.exists == false)
       tags: freemarker
 
-    - name: Install freetype
+    # freetype v2.5.3
+    - name: Test if freetype is already installed
+      win_stat:
+        path: 'C:\openjdk\freetype-2.5.3'
+      register: freetype_installed
+      tags: freetype
+
+    - name: Test if freetype is already installed
+      win_stat:
+        path: 'C:\openjdk\freetype-2.5.3'
+      when: (freetype_installed.stat.exists == false)
```

Comment:
You perform 2 checks here on the directory path setting two different registers.  Not sure this is what you want to do.

Reply:
the second one needs to have its name updated (check if already downloaded)
and path changed to c:\temp\

Reply:
No I do not want it, good catch @jdekonin!